docs/data-modeling/admin-data-model.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Data modeling for admins"
+redirect_from:
+  - /docs/latest/administration-guide/03-metadata-editing
+---
+
+# Data modeling for admins
+
+Let's say you're counting active users. Do you use "count of users", "user count", or "distinct users"? Should we present those counts as "10,000", "10K", or "10000"? And what does "active" really mean, anyway?
+
+When you're [pushing and pulling data](https://www.metabase.com/learn/analytics/push-and-pull) between different teams, you're bound to run into different names, formats, and definitions for the same thing. Enter the **Admin Data Model**: a tiny command centre for all of the business logic living in your Metabase. Don't let the power get to your head.
+
+## Quick start
+
+To start creating a source of truth for your business logic, go to the **Data Model** tab in your **Admin settings**. If you've got more than one database connected to Metabase, click on the database name (for example, "Sample Database") and select another database from the dropdown menu.
+
+You'll find a playground of knobs and buttons that'll let you:
+
+- Set display names and descriptions for [tables](#table-display-name) and [columns](#column-name).
+- Hide outdated or irrelevant [tables](#table-visibility) and [columns](#column-visibility).
+- [Cast a text or number column to a date column](#casting-to-a-specific-data-type) (so you can use the column in datetime expressions, and so on).
+- [Change a filter widget](#changing-the-filter-widget) (for example, from a [search box to a dropdown](#changing-a-search-box-filter-to-a-dropdown-filter)).
+- [Link column values to URLs](#linking-a-column-to-a-url).
+- [Refresh or clear cached filter values](#refresh-or-discard-cached-values).
+- Set up reusable [segments and metrics](./segments-and-metrics.md).
+
+None of the settings in the **Data Model** will affect your database.
```

Comment:
Worth it to move this note up? Or even make it a block quote? And maybe one more clause to explain that the changes only affect how Metabase understands and displays the data.